test/helpers.ts

```diff
@@ -1,8 +1,8 @@
 import { GitProcess, IGitResult } from '../lib'
 
 // NOTE: bump these versions to the latest stable releases
-export const gitVersion = '2.18.0'
-export const gitLfsVersion = '2.5.0'
+export const gitVersion = '2.19.0'
```

Comment:
> when we manually update the respective dependencies?

so this is correct?

Reply:
Yeah, what happens currently is that you'll see these two tests fail because of a version mismatch:

https://github.com/desktop/dugite/blob/398671dec419bdb4f9b29d436d083ecaf6894229/test/fast/git-process-test.ts#L17-L22

https://github.com/desktop/dugite/blob/398671dec419bdb4f9b29d436d083ecaf6894229/test/fast/lfs-test.ts#L17-L22

This is just our way of ensuring we're not accidentally calling your system Git by accident